src/DependencyInjection/Configuration.php

```diff
@@ -365,9 +317,6 @@ public function getConfigTreeBuilder()
             ->booleanNode('direct_publication')
                 ->info($directPublicationInfo)
                 ->defaultValue(false)
-            ->end()
-            ->booleanNode('cache')
```

Comment:
I assume we have to keep this option for now in to order to avoid a BC break without any deprecation.

The default value should be change to false to be consistent and the option will not be used and removed only on 5.x. 
On 4.x it will trigger a deprecation when set.